app/src/main/java/com/example/android/sunshine/sync/SunshineSyncIntentService.java

```diff
@@ -28,6 +28,7 @@ public SunshineSyncIntentService() {
         super("SunshineSyncIntentService");
     }
 
+
```

Comment:
Always be on the lookout for unintended changes like this.  I know this one was a mistake, but if you are doing code formatting, it's best to do that in its own PR and be able to say "Formatting only.  No functional changes." - that makes the PR easy to review.  A line here or there is no big deal, but larger reformatting changes should be done that way.  In this case, you probably removed something, but it's always best to clean up after yourself before committing.